backend/benefit/calculator/rules.py

```diff
@@ -282,9 +286,17 @@ def create_rows(self):
                 end_date=date_ranges[-1].end_date,
                 prefix_text="Koko ajalta",
             )
-            self._create_row(SalaryBenefitSumSubTotalsRow)
+            self._create_row(
+                SalaryBenefitSumSubTotalsRow,
+                start_date=date_ranges[0].start_date,
+                end_date=date_ranges[-1].end_date,
+            )
         else:
-            self._create_row(SalaryBenefitTotalRow)
+            self._create_row(
+                SalaryBenefitTotalRow,
+                start_date=date_ranges[0].start_date,
+                end_date=date_ranges[-1].end_date,
```

Comment:
Same question as in https://github.com/City-of-Helsinki/yjdh/pull/974/commits/cebb439f8dcf54e040740d53a9f442ef1c78580d#r838207292